lib/time.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Pairmotron.Time do
+
+  @doc """
+  Returns true if the year and week passed in are the same week as the current
+  date. Weeks start on Monday. The first week of the year is defined as the first
+  week that contains a Thursday.
+  """
+  def same_week?(year, week, current_date) do
```

Comment:
Would it be worthwhile to change signature to `same_week?/2`? Mostly I mean change the arguments to `{year, week}, current_date` here and clean up all the calls?